public/index.html

```diff
@@ -50,6 +50,7 @@
       ]
       }
     </script>
+  <script src="https://js.stripe.com/v3/"></script>
```

Comment:
can this be a npm package? or else we have to save it to a local file, we done allow loading scripts from 3rd party domains

Reply:
Sure. I've added the @stripe/stripe-js dependency that does this.

Reply:
Thanks